bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -159,12 +193,23 @@ if args.make_template_kde:
                              gl_bandwidth=optbw)
     f_dest.create_dataset("data_kde", data=template_kde)
     f_dest.attrs['stat'] = "template-kde_file"
+    f_dest.attrs['template-file'] = args.attr_filename
+
+
+def new_signal_kde(signal_samples, temp_samples, signal_kde, frac, volume, alp, gl_bandwidth):
```

Comment:
The calculation here looks OK but this function has many arguments which are not used, why are they there?